docs/client/javascript-mono/react-router/_reactRouterPageViews.mdx

```diff
@@ -0,0 +1,56 @@
+## Automatic Page View Tracking
+
+Statsig automatically captures page views when you navigate between routes in your React Router application. This works out of the box with no additional configuration needed.
```

Comment:
this isnt true, you ahve to install the autocapture package.  we should guide people to doing that in the example so they do get this out of the box